examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -0,0 +1,647 @@
+    X_train_preprocessed.std(axis=0),
+    columns=['Coefficient importance'], index=feature_names
+)
+coefs.plot(kind='barh', figsize=(9, 7))
+plt.title('Ridge model, small regularization')
+plt.axvline(x=0, color='.5')
+plt.subplots_adjust(left=.3)
+
+###############################################################################
+# Now that the coefficients have been scaled, we can safely compare them.
+#
+# .. warning::
+#
+#   Why does the plot above suggest that an increase in age leads to a
+#   decrease in wage? Why the :ref:`initial pairplot
+#   <marginal_dependencies>` is telling the opposite?
+#
+# The plot above tells us about dependencies between a specific feature and
+# the target when all other features remain constant, i.e., **conditional
+# dependencies**. An increase of the AGE will induce a decrease
+# of the WAGE when all other features remain constant. On the contrary, an
+# increase of the EXPERIENCE will induce an increase of the WAGE when all
+# other features remain constant.
+# Also, AGE, EXPERIENCE and EDUCATION are the three variables that most
+# influence the model.
+#
+# Checking the variability of the coefficients
+# --------------------------------------------
+#
+# We can check the coefficient variability through cross-validation.
+# If coefficients vary in a significant way changing the input dataset
+# their robustness is not guaranteed, and they should probably be interpreted
+# with caution.
+
+from sklearn.model_selection import cross_validate
+from sklearn.model_selection import RepeatedKFold
+
+cv_model = cross_validate(
+    model, X, y, cv=RepeatedKFold(n_splits=5, n_repeats=5),
+    return_estimator=True, n_jobs=-1
+)
+coefs = pd.DataFrame(
+    [est.named_steps['transformedtargetregressor'].regressor_.coef_ *
+     X_train_preprocessed.std(axis=0)
+     for est in cv_model['estimator']],
+    columns=feature_names
+)
+plt.figure(figsize=(9, 7))
+sns.swarmplot(data=coefs, orient='h', color='k', alpha=0.5)
+sns.boxplot(data=coefs, orient='h', color='cyan', saturation=0.5)
+plt.axvline(x=0, color='.5')
+plt.xlabel('Coefficient importance')
+plt.title('Coefficient importance and its variability')
+plt.subplots_adjust(left=.3)
+
+###############################################################################
+# The problem of correlated variables
+# -----------------------------------
+#
+# The AGE and EXPERIENCE coefficients are affected by strong variability which
+# might be due to the collinearity between the 2 features: as AGE and
+# EXPERIENCE vary together in the data, their effect is difficult to tease
+# apart.
+# To verify this interpretation we plot the variability of the AGE and
+# EXPERIENCE coefficient.
+#
+# .. _covariation:
+
+plt.ylabel('Age coefficient')
+plt.xlabel('Experience coefficient')
+plt.grid(True)
+plt.xlim(-0.4, 0.5)
+plt.ylim(-0.4, 0.5)
+plt.scatter(coefs["AGE"], coefs["EXPERIENCE"])
+_ = plt.title('Co-variations of coefficients for AGE and EXPERIENCE \
```

Comment:
I did not see it in the previous rendering. You should not use the `\` it makes weird spaces.

```python
_ = plt.title('Co-variations of coefficients for AGE and EXPERIENCE'
              'across folds')
```

This breaking should works

Reply:
I quickly edit the file